easybuild/easyblocks/b/bazel.py

```diff
@@ -207,6 +218,7 @@ def sanity_check_step(self):
         }
         custom_commands = []
         if LooseVersion(self.version) >= LooseVersion('1.0'):
-            custom_commands.append("bazel --help")
+            # Avoid writes to $HOME
+            custom_commands.append("bazel --output_user_root=%s --help" % self.output_user_root)
```

Comment:
```suggestion
            custom_commands.append("bazel --output_user_root=%s --help" % self.output_user_root())
```

Reply:
No, this is a property and can be accessed as such.

Reply:
Ah ok, sorry